docs/how-to/find-mysql-backup-file.rst

```diff
@@ -0,0 +1,5 @@
+======================
```

Comment:
I'm not sure this deserves its own docs page? How about in a doc where other paths and such are described?

Reply:
The purpose is different. I think this document serve user who what to find the backup file.

Reply:
Yes, but it's only one sentence - that's why I'm thinking perhaps it can be put with an existing doc?

Reply:
The only document close to it in how-to is https://github.com/canonical/charmed-openstack-upgrader/blob/main/docs/how-to/no-backup.rst

I am thinking if it's a right approach to merge them and rename to `how-to-backup`.

@samuelallan72 How do you think?

Reply:
Updated.